include/zephyr/net/ieee802154_radio.h

```diff
@@ -271,24 +271,29 @@ struct ieee802154_config {
 		} rx_slot;
 
 		/** ``IEEE802154_CONFIG_CSL_PERIOD`` */
-		uint32_t csl_period;
+		uint32_t csl_period; /* in CPU byte order */
 
 		/** ``IEEE802154_CONFIG_CSL_RX_TIME`` */
-		uint32_t csl_rx_time;
+		uint32_t
+			csl_rx_time; /* in microseconds, based on ieee802154_radio_api.get_time() */
```

Comment:
Why such an unexpected line break? IMO better to break the comment if it does not fit.

Reply:
Fixed.